stdlib/OpenBLAS_jll/Project.toml

```
@@ -9,7 +9,7 @@ Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 Artifacts = "56f22d72-fd6d-98f1-02f0-08ddc0907c33"
 
 [compat]
-julia = "1.9"
+julia = "1.10"
```

Unless this is going to be backported, this should be 1.11
```suggestion
julia = "1.11"
```